go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	golang.org/x/exp v0.0.0-20230811145659-89c5cff77bcb
 	golang.org/x/text v0.13.0
 	gopkg.in/yaml.v3 v3.0.1
+	k8s.io/apimachinery v0.27.3
```

Comment:
I don't love this.

Reply:
That's fair. I import it to use the set stuff, but it might be better to just do a map[string]bool like before.